CHANGELOG.md

```diff
@@ -2,335 +2,315 @@
 
 ## [0.27.0.rc1](https://github.com/decidim/decidim/tree/0.27.0.rc1)
 
-DEPRECATION NOTE: The `description` field in the categories admin forms has been removed (this applies to any participatory space using categories). For now it's still available in the database, so you can extract it with the following command in the Rails console:
+### 1. Upgrade notes
+
+As usual, we recommend that you have a full backup, of the database, application code and static files.
+
+To update, follow these steps:
+
+#### 1.1. Update your Gemfile
 
 ```ruby
-Decidim::Category.pluck(:id, :name, :description)
+gem "decidim", "0.27.0.rc1"
+gem "decidim-dev", "0.27.0.rc1"
 ```
 
-In the next version (v0.28.0) it will be fully removed from the database.
+#### 1.2. Run these commands
 
-### Added
+```console
+bundle update decidim
+bin/rails decidim:upgrade
+bin/rails decidim:webpacker:install
+bin/webpack
```

Comment:
@ahukkanen  while working on the Metadecidim update, I found that these couple of commands were necessary, so the app has the service worker on the generated app: [`2414ba0` (#9497)](https://github.com/decidim/decidim/pull/9497/commits/2414ba0eb400e86d2afed6d726d1d6c3a11c8f44). I'm not 100% that this should be done as its: does the `public/sw.js` file should be committed on the generated app or it should be generated by the `assets:precompile` command?

Reply:
My understanding is that it is generated during `assets:precompile` (i.e. when running webpack) automatically, so it should not be committed.

But yes, this should be definitely mentioned in the guidance.

Reply:
Ok, I agree. I've moved it to its own section at 4e18b67d403899beb0f9a40c134aba8a5d620bfe